aws-quickstart/taskcat
# Template url

## Overview

Added try except logic to not exit taskcat because of a bad URL

## Testing/Steps taken to ensure quality

Ran the pre-commit hooks

### Notes

None

## Testing Instructions

How to test this PR Start after checking out this branch (bulleted)
Run this version of taskcat against templates with a badly formed TemplateURL
eg:
{'Fn::Sub': ['https://${S3Bucket}.${S3Region}.{AWS::URLSuffix}/${KeyPrefix}submodules/quickstart-aws-vpc/templates/aws-vpc.template', {'S3Region': {'Fn::Sub': 's3.${AWS::Region}'}, 'S3Bucket': {'Fn::If': ['UsingDefaultBucket', {'Fn::Sub': '${BucketName}-${AWS::Region}'}, {'Ref': 'BucketName'}]}}]}

Note that {AWS::URLSuffix} should be ${AWS::URLSuffix}
